workshops/views.py

```diff
@@ -474,6 +497,13 @@ def task_details(request, task_id):
     return render(request, 'workshops/task.html', context)
 
 
+@login_required
+def task_delete(request, task_id):
```

Comment:
On Wed, Apr 01, 2015 at 02:27:23PM -0700, Piotr Banaszkiewicz wrote:
> > Docstring's should explain what a function (or whatever) does, not
> > who's using it.
> 
> In case of deleting a task, that function redirects to the task's
> event. The redirect is there to help with inline edits from the
> event's page. I think this is unusual enough to at least deserve a
> comment. This redirect is also a part of function's behavior, so it
> may just as well be put in the docstring.

Ah, I agree that it's good to explain any surprising redirects
returned by the function it that function's docstring.  When you said
“where this view is used”, I thought you meant the URL the view was
bound to and any views linking to that URL, not where you'd go after
visiting this view.